PAPI_FAQ.html

```diff
@@ -68,16 +76,16 @@ <h2>The PAPI Library</h2>
 <h2>The PAPI GIT Source Repository</h2>
 
 <strong>Can I browse the source repository on the web?</strong>
-<p>Yes. The latest copy of the PAPI source tree is viewable through a web based source browser accessible from the GitHub repository at:<br /><a href="../../trac/papi/browser">https://github.com/icl-utk-edu/papi</a></p>
+<p>Yes. The latest copy of the PAPI source tree is viewable through a web based source browser accessible from the GitHub repository at:<br /><a href="https://github.com/icl-utk-edu/papi">https://github.com/icl-utk-edu/papi</a></p>
```

Comment:
Currently when I render this, it appears as follows: 
```
Yes. The latest copy of the PAPI source tree is viewable through a web based source browser accessible from the GitHub repository at:
https://github.com/icl-utk-edu/papi
```
I think condensing this to a single line and adding a period to the end of `https://github.com/icl-utk-edu/papi.` would be good. 
```
Yes. The latest copy of the PAPI source tree is viewable through a web based source browser accessible from the GitHub repository at: https://github.com/icl-utk-edu/papi.
```